src/plugins/persistence/couch/plugin.js

```diff
@@ -22,11 +22,15 @@
 
 import CouchObjectProvider from './CouchObjectProvider';
 const NAMESPACE = '';
-const PERSISTENCE_SPACE = 'mct';
+const LEGACY_SPACE = 'mct';
 
 export default function CouchPlugin(options) {
     return function install(openmct) {
         install.couchProvider = new CouchObjectProvider(openmct, options, NAMESPACE);
-        openmct.objects.addProvider(PERSISTENCE_SPACE, install.couchProvider);
+
+        // Unfortunately, for historical reasons, Couch DB produces objects with a mix of namepaces (alternately "mct", and "")
+        // Installing the same provider under both namespaces means that it can respond to object gets for both namespaces.
+        openmct.objects.addProvider(LEGACY_SPACE, install.couchProvider);
+        openmct.objects.addProvider(NAMESPACE, install.couchProvider);
```

Comment:
Interesting! Are we doing this to move away from the 'mct' namespace?

Reply:
Yes, the mct namespace has no special meaning and translating between it and the default namespace just causes confusion. Other changes in this PR mean that the namespace is not persisted with Couch objects anyway, so in principle we could make this configurable in future.